import-manual-mapping.html

```diff
@@ -53,7 +49,7 @@ <h2>Import - Manual Mapping</h2>
                                 <sp-accordion-item label="Load Options">
                                     <div>
                                         <sp-field-label for="import-file-url" required>Transformation file URL</sp-field-label>
-                                        <sp-textfield class="option-field" id="import-file-url" type="url" value="http://localhost:3001/tools/importer/helix-importer-ui/js/sections-mapping/sections-mapping.import.js" disabled></sp-textfield>
+                                        <sp-textfield class="option-field locked" id="import-file-url" type="url" value="http://localhost:3001/tools/importer/helix-importer-ui/js/sections-mapping/sections-mapping.import.js" disabled></sp-textfield>
```

Comment:
When this view (mapping) is enabled, this will no longer overwrite the `import-file-url` value in local-storage?  That is a nice change (that confused me for a while).
Conversely, if someone changes the url in the 'workbench' view, will this value still persist for mapping?

Reply:
Correct. The transformation file used by the mapping tab is hardcoded and will not affect other tabs or vice versa. This should be temporary until mapping is fully integrated and the special mapping script will no longer be required.